celesta-sql/src/main/java/ru/curs/celesta/dbutils/adaptors/ddl/DdlGenerator.java

```diff
@@ -205,13 +205,16 @@ String createTable(TableElement te) {
      * @return Returns String representation of column definition for RDBMS.
      */
     //TODO:Must be defined in single place
-    final String columnDef(Column c) {
+    final String columnDef(Column<?> c) {
+        @SuppressWarnings("unchecked")
+        final Class<? extends Column<?>> cClass = (Class<Column<?>>) c.getClass();
```

Comment:
I don't know how to make it without the excplicit type conversion.

Reply:
Doesn't look a big problem to me. To avoid `@SuppressWarnings`,  we might just use `Class<? extends Column>` all the way down instead of `Class<? extends Column<?>>`...